### PR TITLE
Player.DropItems() & Player.ClearInventory() modification

### PR DESCRIPTION
**Pull request by Thundermaker300**
* Added Player.DropItems() as a quick way to drop all items as opposed to clearing them.
* Modified Player.ClearInventory() to use Inventory.Clear() instead of clearing the items from the inventory manually.

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1146,7 +1146,12 @@ public void ResetInventory(List<ItemType> newItems)
         /// <summary>
         /// Clears the player's inventory.
         /// </summary>
-        public void ClearInventory() => Inventory.items.Clear();
+        public void ClearInventory() => Inventory.Clear();
```
Comment (iRebbok) [inline Exiled.API/Features/Player.cs:1149]: `Inventory::Clear()` also affects the dropping of `AmmoBox`. I think you shouldn't have changed it.

**Review by iRebbok — CHANGES_REQUESTED** [review]
Why did you move the `Exiled.nuspec` file? You shouldn't have changed its location.

File: Exiled.API/Features/Player.cs [hunk 2]
```diff
@@ -1146,7 +1146,12 @@ public void ResetInventory(List<ItemType> newItems)
         /// <summary>
         /// Clears the player's inventory.
```
Comment (iRebbok) [inline Exiled.API/Features/Player.cs:1147]: Update the summary, like
```
Clears the player's inventory. Includes ammo and items.
```

**Comment by Thundermaker300** [comment]
I deleted the nuspec file?? I didn't even touch it.. Okay, I'll make the changes

**Comment by Thundermaker300** [comment]
I think I fixed it now.

**Review by iRebbok — APPROVED** [review]

**Pull request merged**